Preparing the specification for LDML47 release (#1007)
unicode-org/message-format-wg: spec/functions/datetime.md

- `numberingSystem`
  - valid [Unicode Number System Identifier](https://unicode.org/reports/tr35/tr35.html#UnicodeNumberSystemIdentifier)

> > [!IMPORTANT]
> > The following _option_ and its values are **Draft** and proposed for
> > inclusion in a future release of this specification but have not yet been
> > finalized.
> > If accepted, implementations could be REQUIRED to make this _option_
> > available in the functions `:datetime`, `:date`, and `:time`.

Reviewer: Everything here is draft, so we no longer need to note this as anything special.

```suggestion
The following _option_ and its values are REQUIRED to be available on
the functions `:datetime`, `:date`, and `:time`.
```
